src/Symfony/Component/Security/Http/Logout/LogoutUrlGenerator.php

```diff
@@ -48,21 +48,8 @@ public function __construct(RequestStack $requestStack = null, UrlGeneratorInter
      * @param CsrfTokenManagerInterface|null $csrfTokenManager A CsrfTokenManagerInterface instance
      * @param string|null                    $context          The listener context
      */
-    public function registerListener($key, $logoutPath, $csrfTokenId, $csrfParameter, CsrfTokenManagerInterface $csrfTokenManager = null/*, $context = null*/)
+    public function registerListener($key, $logoutPath, $csrfTokenId, $csrfParameter, CsrfTokenManagerInterface $csrfTokenManager = null, string $context = null)
```

Comment:
Related PR to add the typehint in 3.3: https://github.com/symfony/symfony/pull/22757